src/common/models/Ratio.ts

```diff
@@ -0,0 +1,38 @@
+import { AssetInfo } from '@ergolabs/ergo-sdk/build/main/entities/assetInfo';
+
+import { parseUserInputToFractions, renderFractions } from '../../utils/math';
+export class Ratio {
```

Comment:
Are you sure about this structure?
Why didn't you place all of its functionality to `Currency`?
I see the violation of DRY the principle in that structure.

Reply:
Currency structure depends on asset decimals. We need separate structure for prices